.github/workflows/dart_frog_auth.yaml

```diff
@@ -18,11 +18,10 @@ on:
 
 jobs:
   build:
-    uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/dart_package.yml@v1
+    uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/dart_package.yml@v1.13.0
```

Comment:
v1 and v1.13.0 point to the same commit.

Reply:
```suggestion
    uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/dart_package.yml@v1
```